de.ifw_dresden.nagstamon.appdata.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>de.ifw_dresden.nagstamon</id>
+	<metadata_license>GPLv2</metadata_license>
+	<name>Nagstamon</name>
+	<summary>Nagstamon is a status monitor for the desktop.</summary>
+	<description>
+		<p>Nagstamon is a status monitor for the desktop. It connects to multiple Nagios, Icinga, Opsview, Centreon, </p>
+           <p>Op5 Monitor/Ninja, Check_MK Multisite and Thruk monitoring servers. It resides in systray, as a floating </p>
+           <p>statusbar or fullscreen at the desktop showing a brief summary of critical, warning, unknown, unreachable </p>
+           <p>and down hosts and services. It pops up a detailed status overview when being touched by the mouse pointer. </p>
+           <p>Connections to displayed hosts and services are easily established by context menu via SSH, RDP, VNC or </p>
+           <p>any self defined actions. Users can be notified by sound. Hosts and services can be filtered by category </p>
+           <p>and regular expressions.</p>
+	</description>
+	<url type="homepage">https://nagstamon.ifw-dresden.de/</url>
+	<url type="help">https://nagstamon.ifw-dresden.de/docs/</url>
```

Comment:
You could add bugtracker url here

Reply:
I've added the url pointing to the project's github issues page. Is that ok?